tests/unit/resources/test_policies.py

```diff
@@ -197,6 +197,21 @@ def test_create_single_rules(self, mock_get, mock_post):
         mock_post.assert_called_once_with('/policies/policy12345/rules?replace_all_rules=False', [rules],
                                           custom_headers=None)
 
+    @mock.patch.object(Connection, "delete")
+    @mock.patch.object(Connection, "get")
+    def test_delete_rule(self, mock_get, mock_delete):
+        mock_delete.return_value = None, [{'object_id': 'policy12345'}]
+        resources_data = {"rules": [], "name": "name", "id": "policy12345"}
+        mock_get.return_value = {'policy': resources_data}
+        policy_data = {"rules": [{"frequency": 5, "id": "12345", "retention": 1}], "name": "name",
+                       "id": "policy12345"}
```

Comment:
in all other tests id(s) are typically just a group of 5 numbers ( like 12345, or 67890 ).  I don't think I matters here.  Just an observation in case someone else knows a reason not to do it this way.

Reply:
@svt-kmcdaniel @sijeesh 
Please let us know if we can have ID like "policy12345" or we need to update to "67890" any combination of 5 digit numbers.

Reply:
As it's a test case it doesn't matter much but to align with other test cases it's better to use the same kind of ids we used for other test cases.

Reply:
Thanks @sijeesh , @svt-kmcdaniel 
Updated the policy id to '67890'